bootstrap/shell/shell.go

```diff
@@ -197,6 +197,12 @@ func (s *Shell) Run(command string, arg ...string) error {
 	return s.RunWithoutPrompt(command, arg...)
 }
 
+func (s *Shell) RunWithInput(input string, command string, arg ...string) error {
```

Comment:
Agreed, but I couldn't figure out how to setup a pipe from the output of one shell command to another which would pump and close correctly, so it seemed a simple first step to do with strings.

An ideal implementation might tell the shell to run the two processes piped together, and show a fancy "command1 | command2" Promptf.

Reply:
Yeah, for now we can just `strings.NewReader(gitCommitOutput)` to turn the `string` output from the previous command into an `io.Reader` input to the next command.

But good idea, something like `shell.RunPipeline(…)`

Reply:
`strings.NewReader(gitCommitOutput)` is what I first tried, but I couldn't get the command to finish and return from the `shell.Run` call, I assumed because it wasn't being closed... but I'm not sure!